Update composer.json

de-sec opened the pull request:
Unexpected quotation mark breaks the Plugins Page of Kimai (HTTP 500; PHP Array Issue)

Commit:
Update composer.json

jens-2305 reviewed (approved):
thanks

kevinpapst reviewed (approved):
@jens-2305 looks good to me, what about merging?

Commit:
Migration to Kimai2 V2

Migrated Code and added Workaround for ExportContext.php in Readme

de-sec commented:
Additional Update to Migrate to Kimai2 V2 (including Workaround for ExportContext.php from Kimai2 V1 in Readme)

kevinpapst commented:
DON'T recommend this please.

This is what you are looking for: 
https://github.com/kimai/kimai/blob/main/src/Pdf/PdfContext.php

Commit:
Migration from ExportContext to PdfContext.php

As recommended by @kevinpapst

Commit:
Update README.md

Migration ExportContext to PdfContext as suggested by @kevinpapst in https://github.com/jens-2305/ZipProjectRendererBundle/pull/2#issuecomment-1743067846

de-sec commented:
> DON'T recommend this please.
> 
> This is what you are looking for: https://github.com/kimai/kimai/blob/main/src/Pdf/PdfContext.php

@kevinpapst Thanks for the Feedback, edited files according to your recommendations

kevinpapst reviewed (commented):
@de-sec better bump to version 1.0, as a new Kimai major version is needed

@jens-2305 any plans to merge the changes / maintain that package? If not, I will link the repo of de-sec from the Store.

Commit:
Update composer.json

Bump Version

kevinpapst commented on CHANGELOG.md:
```diff
@@ -3,3 +3,7 @@
   
 intitial Version
+
+## 0.9
+
+migration to Kimai2 v2
```
```suggestion
## 1.0

migration to Kimai 2.0
```

de-sec replied:
Done!

Commit:
Update CHANGELOG.md
